src/app/modules/item/components/thread-message/thread-message.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Input, OnChanges, OnDestroy } from '@angular/core';
+import { IncomingThreadEvent } from '../../services/threads-inbound-events';
+import { UserSessionService } from '../../../../shared/services/user-session.service';
+import { ReplaySubject, combineLatest } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+@Component({
+  selector: 'alg-thread-message[event]',
+  templateUrl: './thread-message.component.html',
+  styleUrls: [ './thread-message.component.scss' ],
+})
+export class ThreadMessageComponent implements OnChanges, OnDestroy {
+  @Input() event!: IncomingThreadEvent;
+
+  event$ = new ReplaySubject<IncomingThreadEvent>(1);
+  isMessageCreatedByCurrentUser$ = combineLatest([
+    this.userSessionService.userProfile$,
+    this.event$,
+  ]).pipe(
+    map(([ currentUser, event ]) => currentUser.groupId === event.createdBy),
+  );
+
```

Comment:
I think it would have be better to pass the current user (from parent component) and then to work without Rx at the level of this component... (so that we do not have 100 messages each subscribing to the profile)
Anyway, we can change that later.